ert3/storage/_storage.py

```diff
@@ -92,16 +96,18 @@ def _init_experiment(
         url=f"{_STORAGE_URL}/experiments/{exp_id}/ensembles",
         json={
             "parameter_names": list(parameters),
-            "response_names": [],
+            "response_names": list(responses),
             "size": ensemble_size,
+            # Add name to be displayed in Webviz - ert
```

Comment:
I suggest we remove this comment, as if we want to convey this it should be done in code.